generators/generator-bot-core-assistant/generators/app/templates/dialogs/BotTourDialog/BotTourDialog.dialog

```diff
@@ -34,17 +34,79 @@
       "intent": "DetailsCard",
       "actions": [
         {
-          "$kind": "Microsoft.SendActivity",
+          "$kind": "Microsoft.ChoiceInput",
           "$designer": {
-            "id": "Z5JucQ"
+            "id": "8kK1VI"
+          },
+          "defaultLocale": "en-us",
+          "disabled": false,
+          "maxTurnCount": 3,
+          "alwaysPrompt": false,
+          "allowInterruptions": true,
+          "unrecognizedPrompt": "",
+          "invalidPrompt": "",
+          "prompt": "${ChoiceInput_Prompt_BotTourDetails()}",
+          "choiceOptions": {
+            "includeNumbers": true,
+            "inlineOrMore": ", or ",
+            "inlineOr": " or ",
+            "inlineSeparator": ", "
+          },
+          "choices": [
```

Comment:
I trust this is valid, but it seems a little duplicative since the actions in the returned card already contain the results per se. Am I thinking about this wrong?

Reply:
The original "prompt" was just a SendActivity with the card, where the card actions where separate intents (this logic remains in how we go from the Overview -> Detail card). This version replaces that logic with a multi-choice prompt with the display of those choices hidden. Instead, this has the old adaptive card but the data returned is a string that matches the expected choices listed here.

Reply:
I prefer this approach to handling different intents because in this case, the intents are just dummy data and there is no point in populating a new bot with a more complicated setup that they will just want to remove anyways.